src/allProjects.json

```diff
@@ -9267,6 +9813,20 @@
     "slug": "wings",
     "ticker": "WINGS"
   },
+  {
+    "coinmarketcapId": "wink-tronbet",
+    "id": "235738",
+    "name": "Wink",
+    "slug": "wink-tronbet",
+    "ticker": "WIN"
```

Comment:
I saw that this was missing the price in the insights: 
<img width="636" alt="image" src="https://user-images.githubusercontent.com/6518376/62625669-5b207a00-b92e-11e9-86cf-226be0e45d8d.png">